<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Painter</class>
 <widget class="QWidget" name="Painter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEditFont">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBrush">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_savePix">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>128</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回计算器</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
